usaspending_api/references/management/commands/load_naics.py

```diff
@@ -40,8 +40,12 @@ def populate_naics_fields(ws, naics_year):
         if not row[0].value:
             break  # Reads file only until a blank line
 
-        naics_code = row[0].value
-        naics_desc = row[1].value
+        try:
+            naics_code = int(row[0].value)
+        except:
+            logging.warning('could not parse to string: %s', row[0].value)
```

Comment:
What does the try-except block do for the script? if it fails the type cast it will produce a python exception below since `naics_code` won't be defined. If you want to skip you will need to add a `continue` here. But I would discourage that. if int() is failing on any entries, perhaps you should use .strip() like you do for the description.

Reply:
For validation purposes, the idea is to stop the script. The reasoning being that if the NAICS code cannot be parsed into an integer, the source file would need to be reviewed for integrity/security.

Reply:
If that is the case, you should raise an exception with a helpful and useful message in the except block to explicitly stop the script there.